test/unit/modules/rounds.js

```diff
@@ -369,29 +369,22 @@ describe('rounds', () => {
 	describe('__private.sumRound', () => {
 		var sumRound;
 		var stub;
-		var scope = { round: 1 };
 
 		before(done => {
 			sumRound = get('__private.sumRound');
 			done();
 		});
 
-		describe('when summedRound query is successful', () => {
-			before(done => {
-				var rows = [
-					{
-						rewards: [1.001, 2, 3],
-						fees: 100.001,
-						delegates: ['delegate1', 'delegate2', 'delegate3'],
-					},
-				];
-				stub = sinon.stub(db.rounds, 'summedRound').resolves(rows);
-				done();
-			});
-
-			after(() => {
-				return stub.restore();
-			});
+		describe('when last block is genesis block', () => {
+			const scope = {
+				round: 1,
```

Comment:
As stated above, `round` should not be passed as scope to sumRound, it should always calculate `round` from the `height` of the block.

Reply:
As above, can be done as a separate issue.